automation.md

```diff
@@ -0,0 +1,102 @@
+```
+According to the chrome startup parameter description:
+```
+--remote-debugging-pipe: Enables remote debug over stdio pipes [in=3, out=4]. Optionally, specifies the format for the protocol messages, can be either "JSON" (the default) or "CBOR"
+```
+
+How to use CDP, you can get started from [this link](https://github.com/aslushnikov/getting-started-with-cdp/blob/master/README.md).
+More crhome startup parameters can be found in the official documentation, or [here](https://github.com/GoogleChrome/chrome-launcher/blob/master/docs/chrome-flags-for-tools.md).
+- Native messaging
+Extensions and apps can exchange messages with native applications using an API that is similar to the other [message passing APIs](https://developer.chrome.com/docs/apps/nativeMessaging/messaging). Native applications that support this feature must register a native messaging host that knows how to communicate with the extension. Chrome starts the host in a separate process and communicates with it using standard input and standard output streams.
+
+More about Native Messaging, please visit [this link](https://developer.chrome.com/docs/apps/nativeMessaging/messaging).
+
+#  SAP Automation
+SAP itself provides script automation, so most RPA products automate SAP applications based on the automation technology provided by SAP itself.
+
+- Automated supplementary technology
+In addition to the above native automation technologies, the following automation technologies are generally used to achieve hybrid automation
+  - Image recognition, shortcut keys
+  - OCR
+  - Screen word picking
+  - computer vision
+  - Windows Native API
+
+# What Clicknium Do
+Utilizing the aforementioned automation technologies, Clicknium offers development tools that allow users to: 
+- capture UI elements and store them as locators or selectors.
+- a single Python API to automate all application kinds.
+- Higher efficiency in writing code with locator IntelliSense.
+
+In addition to utilizing the aforementioned automation technologies, Clicknium also performs other enhancements: 
```

Comment:
In addition to the  automation technologies above,  Clicknium also performs other enhancements: